automation/patch_igz/patch_remote.py

```diff
@@ -224,12 +254,13 @@ def _push_docker_images(self, built_images):
         logger.info(f"Pushing mlrun docker images: {built_images}")
 
         for image in built_images:
+            cmd = [
+                "docker",
+                "push",
+                image,
+            ]
             self._exec_local(
-                [
-                    "docker",
-                    "push",
-                    image,
-                ],
+                cmd,
                 live=True,
             )
```

Comment:
Not that it's wrong, but is there a reasoning for this change?

Reply:
fixed